<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlphaBetaHHDlg</class>
 <widget class="QDialog" name="AlphaBetaHHDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>610</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>8</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>HH alpha,beta Formalism</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="5" column="5" colspan="2">
    <widget class="QComboBox" name="LUCombo">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>Direct Functions</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LookupTables</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Vrev</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="mkaE">
     <property name="text">
      <string>1.28</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLineEdit" name="msaE">
     <property name="text">
      <string>-4.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="5">
    <widget class="QLabel" name="label_31">
     <property name="text">
      <string>hVb</string>
     </property>
    </widget>
   </item>
   <item row="9" column="5">
    <widget class="QLabel" name="label_38">
     <property name="text">
      <string>ha type</string>
     </property>
    </widget>
   </item>
   <item row="15" column="3">
    <widget class="QLabel" name="label_28">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>nS</string>
     </property>
    </widget>
   </item>
   <item row="15" column="6">
    <widget class="QLineEdit" name="hVbE">
     <property name="text">
      <string>-25.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>gMax</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Activation</string>
     </property>
    </widget>
   </item>
   <item row="14" column="6">
    <widget class="QLineEdit" name="hkbE">
     <property name="text">
      <string>4.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="6" colspan="2">
    <widget class="QComboBox" name="haFuncCombo">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>k*V/(exp(V)-1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*exp(V)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*1/(1+exp(V))</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="13" column="6" colspan="2">
    <widget class="QComboBox" name="hbFuncCombo">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <item>
      <property name="text">
       <string>k*V/(exp(V)-1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*exp(V)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*1/(1+exp(V))</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLineEdit" name="mVaE">
     <property name="text">
      <string>-52.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QLabel" name="MethodL">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Method</string>
     </property>
    </widget>
   </item>
   <item row="16" column="6">
    <widget class="QLineEdit" name="hsbE">
     <property name="text">
      <string>-5.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QLabel" name="label_40">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Channel assignments</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>mkb</string>
     </property>
    </widget>
   </item>
   <item row="16" column="7">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>mVa</string>
     </property>
    </widget>
   </item>
   <item row="11" column="5">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>hVa</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QComboBox" name="mbFuncCombo">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>k*V/(exp(V)-1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*exp(V)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*1/(1+exp(V))</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="8">
    <widget class="AssignmentWidget&lt;CurrentAssignment&gt;" name="assignments">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>25</number>
     </attribute>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="QLineEdit" name="msbE">
     <property name="text">
      <string>5.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="7">
    <widget class="QLabel" name="label_35">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="VrevE">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>10.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="gMaxE">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>10.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="6">
    <widget class="QLineEdit" name="hkaE">
     <property name="text">
      <string>0.128</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="5">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>hka</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0" colspan="2">
    <widget class="QLabel" name="label_36">
     <property name="text">
      <string>ma type</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_18">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="label_37">
     <property name="text">
      <string>mb type</string>
     </property>
    </widget>
   </item>
   <item row="11" column="6">
    <widget class="QLineEdit" name="hVaE">
     <property name="text">
      <string>-48.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="label_13">
     <property name="text">
      <string>1/ms</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Maximum</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Exponent</string>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="label_21">
     <property name="text">
      <string>msb</string>
     </property>
    </widget>
   </item>
   <item row="12" column="7">
    <widget class="QLabel" name="label_32">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="10" column="7">
    <widget class="QLabel" name="label_29">
     <property name="text">
      <string>1/ms</string>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="QLineEdit" name="mVbE">
     <property name="text">
      <string>-25.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="5">
    <widget class="QLabel" name="label_39">
     <property name="text">
      <string>hb type</string>
     </property>
    </widget>
   </item>
   <item row="12" column="5">
    <widget class="QLabel" name="label_33">
     <property name="text">
      <string>hsa</string>
     </property>
    </widget>
   </item>
   <item row="17" column="5" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="12" column="6">
    <widget class="QLineEdit" name="hsaE">
     <property name="text">
      <string>-18.0</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="mExpoE">
     <property name="text">
      <string>3</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>mka</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4" rowspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="label_20">
     <property name="text">
      <string>mVb</string>
     </property>
    </widget>
   </item>
   <item row="14" column="7">
    <widget class="QLabel" name="label_34">
     <property name="text">
      <string>1/ms</string>
     </property>
    </widget>
   </item>
   <item row="11" column="7">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="8" column="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Exponent</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <widget class="QLineEdit" name="hExpoE">
     <property name="text">
      <string>1</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>1/ms</string>
     </property>
    </widget>
   </item>
   <item row="16" column="3">
    <widget class="QLabel" name="label_26">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_19">
     <property name="text">
      <string>msa</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QLabel" name="label_17">
     <property name="text">
      <string>mV</string>
     </property>
    </widget>
   </item>
   <item row="14" column="5">
    <widget class="QLabel" name="label_25">
     <property name="text">
      <string>hkb</string>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="QLineEdit" name="mkbE">
     <property name="text">
      <string>1.4</string>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="5">
    <widget class="QLabel" name="label_24">
     <property name="text">
      <string>hsb</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2" colspan="2">
    <widget class="QComboBox" name="maFuncCombo">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>k*V/(exp(V)-1)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*exp(V)</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>k*1/(1+exp(V))</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="5" colspan="2">
    <widget class="QLabel" name="label_15">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Inactivation</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLineEdit" name="leLabel">
     <property name="placeholderText">
      <string>Label...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="3">
    <widget class="QLabel" name="HHDlgLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssignmentWidget&lt;CurrentAssignment&gt;</class>
   <extends>QTableWidget</extends>
   <header location="global">AssignmentWidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>assignments</tabstop>
  <tabstop>gMaxE</tabstop>
  <tabstop>VrevE</tabstop>
  <tabstop>mExpoE</tabstop>
  <tabstop>maFuncCombo</tabstop>
  <tabstop>mkaE</tabstop>
  <tabstop>mVaE</tabstop>
  <tabstop>msaE</tabstop>
  <tabstop>mbFuncCombo</tabstop>
  <tabstop>mkbE</tabstop>
  <tabstop>mVbE</tabstop>
  <tabstop>msbE</tabstop>
  <tabstop>LUCombo</tabstop>
  <tabstop>hExpoE</tabstop>
  <tabstop>haFuncCombo</tabstop>
  <tabstop>hkaE</tabstop>
  <tabstop>hVaE</tabstop>
  <tabstop>hsaE</tabstop>
  <tabstop>hbFuncCombo</tabstop>
  <tabstop>hkbE</tabstop>
  <tabstop>hVbE</tabstop>
  <tabstop>hsbE</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AlphaBetaHHDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AlphaBetaHHDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
